### PR TITLE
DO-1545: Prerender Fargate Readme update

### PR DESCRIPTION
**Pull request by krishanthisera**
### Prerender Fargate Readme update
------

**Description of the proposed changes**  

* Prerender Fargate Readme update

**Screenshots (if applicable)**  

* N/A

**Other solutions considered (if any)**  

* N/A

🛈  When you've finished leaving feedback, please add a final comment to the PR tagging the author, letting them know that you have finished leaving feedback

### CODE REVIEW

File: packages/prerender-fargate/README.md [hunk 1]
```diff
@@ -1,18 +1,113 @@
...
-- `tokenList`: List of tokens to accept as authentication
-- `certificateArn`: Certificate arn to match the domain
-- `desiredInstanceCount`: Number of Prerender instances to run (default 1)
-- `maxInstanceCount`: Maximum number of Prerender instances to run (default 2)
-- `instanceCPU`: CPU to allocate to each instance (default 512)
-- `instanceMemory`: Amount of memory to allocate to each instance (default 1024)
-- `enableRedirectCache`: Cache 301 and 302 responses, too (default false)
+# PrerenderFargate Construct
+
+The PrerenderFargate construct sets up an AWS Fargate service to run a [Prerender] service in an ECS Fargate cluster.
```
Comment (TheOrangePuff) [inline packages/prerender-fargate/README.md:3]: Need to add the link to prerender.io :sweat_smile:

File: packages/prerender-fargate/README.md [hunk 2]
```diff
@@ -1,18 +1,113 @@
...
+
+- Whether to enable caching of HTTP redirects.
+
+### `enableS3Endpoint` (boolean, optional)
+
+- Whether to enable the S3 endpoint for the VPC.
+
+### `tokenUrlAssociation` (PrerenderTokenUrlAssociationOptions, optional)
+
+- Configuration for associating tokens with specific domain URLs. During the recaching process, these tokens will be used to validate the request.
```
Comment (TheOrangePuff) [inline packages/prerender-fargate/README.md:73]: This might need a little more explanation

File: packages/prerender-fargate/README.md [hunk 3]
```diff
@@ -1,18 +1,113 @@
...
+
+- The amount of memory to allocate to each Fargate instance.
+
+### `enableRedirectCache` (string, optional)
+
+- Whether to enable caching of HTTP redirects.
+
+### `enableS3Endpoint` (boolean, optional)
+
+- Whether to enable the S3 endpoint for the VPC.
```
Comment (TheOrangePuff) [inline packages/prerender-fargate/README.md:69]: A link to the AWS docs here would be good

**Review by TheOrangePuff — CHANGES_REQUESTED** [review]
So much better than it was before :heart_eyes: Just a few minor requests from me

File: packages/prerender-fargate/README.md (hunk 1 above)
Comment (krishanthisera) [reply packages/prerender-fargate/README.md:3]: I have included in the Acknowledgments section

File: packages/prerender-fargate/README.md (hunk 2 above)
Comment (krishanthisera) [reply packages/prerender-fargate/README.md:77]: There will be some changes, so I will let @crispy101 to update all together

**Review by crispy101 — APPROVED** [review]
LGTM

**Review by TheOrangePuff — APPROVED** [review]

**Pull request merged**